tests/RuleSetTest.php

```diff
@@ -884,6 +885,9 @@ private static function assertPHPUnitVersionIsLargestAllowed($setName, $ruleName
 
         foreach ($fixer->getConfigurationDefinition()->getOptions() as $option) {
             if ('target' === $option->getName()) {
+                if ($option instanceof DeprecatedFixerOption) {
+                    static::markTestSkipped(sprintf('The fixer "%s" has option "target" deprecated.', $fixer->getName()));
+                }
```

Comment:
shouldn't the test still continue with possible other options, like have a `continue` here?

Reply:
No need to, the whole `foreach` loop is to find `target` option (it does `break` after finding it).

Reply:
but the `markTestSkipped` will throw an exception as soon as it finds a deprecated option and so it will no longer checks for other option candidates that might come after it